db/migrate/20230815130345_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :author_id
+      t.integer :post_id
```

Comment:
:red_circle: [REQUIRED] Similar to the previous example, you can add foreign key constraints to ensure that the author_id and post_id fields in the comments table reference valid id values in their respective tables (authors and posts).

Here's how you can do that:

```
create_table :comments do |t|
    t.references :author, foreign_key: true # Add foreign key constraint for author
    t.references :post, foreign_key: true   # Add foreign key constraint for post
    t.text :text

    t.timestamps
  end
```
In this code, the lines t.references :author, foreign_key: true and t.references :post, foreign_key: true add foreign key constraints to the author_id and post_id fields, respectively. These constraints ensure that the values in these fields reference valid records in the authors and posts tables, maintaining the integrity of your data.